code/RedirectedURLAdmin.php

```diff
@@ -39,7 +39,7 @@ class RedirectedURLAdmin extends ModelAdmin {
 	 * @return array Map of model class names to importer instances
 	 */
 	public function getModelImporters() {
-		$importer = new CsvBulkLoader("RedirectedURL");
+		$importer = new RedirectedURLCsvBulkLoader("RedirectedURL");
 		$importer->duplicateChecks = array(
 			'FromBase' => array('callback' => 'findByFrom'),
```

Comment:
I see we have a callback for finding duplicates, is this now not needed?

Reply:
Removed. I think that's not needed as we use `ValidationException` to handle duplicates